### DISCUSSION TITLE
Can i use this model to create a windows application that have speech recognition instead of a server?

### DISCUSSION THREAD

**Discussion by ali-h2010** [NVIDIA-NeMo/NeMo#2063] [Q&A]
I noticed that the system is well documented but there is no section about how can we select what we can install such as a specific language and what is the installation size. I noticed in colab that the instillation size is over 700 mb. Does that mean the system is designed to work on a server only?

**Comment by okuchaiev** [comment]
no, you can run various models locally. However, at the moment we support only Ubuntu.

**Reply by ali-h2010** [reply]
thanks for the update. 
Is there a tight dependency on the operating system? I though Python runtime and its libraries are the only dependencies.

**Reply by okuchaiev** [reply]
no, and I think it could be made to work. However, this isn't something we test and support at the moment.
